### DISCUSSION TITLE
Missing manual workflow trigger in list view

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#15209] [Feature improvement]
Would it bring some disadvantages adding Manual Workflow Trigger in list view? It seems to be useful in some use cases

Jonny

**Comment by rskrzypczak** [comment]
Performance.
Each such workflow should be verified per record, so mass starting the workflow can be confusing at times as different workflow actions may be available for different records.

**Reply by JohnMaverick** [reply]
Clear. Thanks

**Reply by garryatbrighton** [reply]
But... it is easy to trigger such workflow by changing some field value in mass edit.... Please remember that this can be controlled from Admin level. I think it will be OK to add WF to list view. Just shorter way in many scenarios

> Performance. Each such workflow should be verified per record, so mass starting the workflow can be confusing at times as different workflow actions may be available for different records.

**Comment by s2b-git** [comment]
This is workflow dialog we used to use in VT.
Very helpful I must say

![image](https://user-images.githubusercontent.com/84110093/218004590-03ce1ac6-3fad-4d26-8108-67851ae82032.png)
